examples/sample-operator/README.md

```diff
@@ -0,0 +1,28 @@
+# Operator reference implementation
+
+ `sample-operator` controls deployment of a simple, not customized nginx HTTP server.
+ 
+ ## Deployment
+ `sample-operator` is deployed in `kubevirt` namespace and deployment manifests will create it, if it's not present.
+ 
+ To deploy the operator execute following command:
+ ```shell script
+kubectl apply -f deploy/operator.yaml
+```
+
+Deployment of the HTTP server is controlled by the presence of a `sampleconfig` CR; to deploy it, execute:
+```shell script
+kubectl apply -f deploy/crd/sampleconfig_types.go
+```
+
+The HTTP service listens on port 8081, which can be forwarded locally:
+```shell script
+kubectl -n kubevirt port-forward service/http-server 8081:8081
```

Comment:
What do you think about adding this in the operator as any other resource we deploy?